rockchip: rk3588: Increase the MMU600PHP QOS to 4 for USB3_2

Ken-Vamrs opened the pull request:
This patch increases the MMU600PHP QOS from default 0 to 4 to get better transmission performance for USB3_2 & SATA0-2 & GMAC0-1.


Change-Id: I3a51483e41310c2abbca93a6a9c6e51d5ce50841

wuliangfeng committed:
rockchip: rk3588: Increase the MMU600PHP QOS to 4 for USB3_2

This patch increases the MMU600PHP QOS from default 0
to 4 to get better transmission performance for USB3_2
& SATA0-2 & GMAC0-1.

Signed-off-by: William Wu <william.wu@rock-chips.com>
Change-Id: I3a51483e41310c2abbca93a6a9c6e51d5ce50841

Ken-Vamrs commented:
RK redmine: https://redmine.rock-chips.com/issues/510020

RadxaYuntian commented:
少了0001-phy-rockchip-naneng-combophy-adjust-Rx-squelch-for-R.patch

Ken-Vamrs commented:
> 少了0001-phy-rockchip-naneng-combophy-adjust-Rx-squelch-for-R.patch

这个patch是kernel的，而且已经有合入的了

RadxaYuntian reviewed (changes requested):
看上去还有启动问题：https://redmine.rock-chips.com/issues/510020#change-5155839

Ken-Vamrs committed:
rockchip: rk3588: write MMU600PHP registers after running spl

Fix crash when writing to registers in spl stage.

U-Boot SPL board init
U-Boot SPL 2017.09-g875ab5399d8-220414-dirty #ken (Sep 25 2024 - 15:10:43)
SPL Hotkey: ctrl+`
"Error" handler, esr 0xbe000011
ELR:     810
LR:      bc4
ESR:     be000011 (ec=47)
x 0: 0000000000000000 x 1: 00000000fdf3a608
x 2: 0000000000000020 x 3: 000000000000000d
x 4: 00000000000012b4 x 5: 0000000000000000
x 6: 00000000000012b0 x 7: 0000000000000002
x 8: 00000000000012ac x 9: 000000000000000c
x10: 0000000003f5fd1c x11: 0000000000038420
x12: 00000000000012b0 x13: 0000000003f5fd6c
x14: 0000000000038420 x15: 0000000000038420
x16: 00000000f7000000 x17: 00000000ff140000
x18: 0000000003f5fe30 x19: 0000000003f5fe30
x20: 0000000000000000 x21: 00000000ff000020
x22: 00000000ff001000 x23: 00000000ff000000
x24: 00000000534e4b52 x25: 0000000053534b52
x26: 00000000ff000000 x27: 00000000ff000000
x28: 00000000ff000000 x29: 0000000003f5fe10

Call trace:
  PC:   [< 00000810 >]
  LR:   [< 00000bc4 >]

Stack:
        [< 00000810 >]

Copy info from "Call trace..." to a file(eg. dump.txt), and run
command in your U-Boot project: ./scripts/stacktrace.sh dump.txt spl

Resetting CPU ...

Signed-off-by: Ken Wang <ken@radxa.com>
